tests/test_config.py

```
@@ -0,0 +1,171 @@
+            "chart.name",
+        ],
+    )
+    def test_invalid_chart_names(self, name: str) -> None:
+        """Test that invalid chart names do not match the pattern."""
+        assert CHART_NAME_PATTERN.match(name) is None
+
+
+class TestConfig:
+    """Tests for Config class."""
+
+    def test_get_github_token(self, mock_env_vars: dict) -> None:
```

Ruff (0.14.13)

Unused method argument: `mock_env_vars`

(ARG002)